### PR TITLE
Add my username

### PR DESCRIPTION
**Pull request by nikkif24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ GitHub Handle | Fun Fact
 @kochub       | I love reading books
 @AndrewParkerTechnicalTester | The table formatting still works regardless of the alignment of the raw text
 @ralamo       | I like making pie.
-@ogriff17     | This is me making changes to the ReadMe file.
\ No newline at end of file
+@ogriff17     | This is me making changes to the ReadMe file.
+@nikkif24 |I figured it out?!?
```
Comment (sguthals) [inline README.md:14]: Great job!!!! :D

**Comment by sguthals** [comment]
Welcome!

**Comment by nikkif24** [comment]
This email made my day!! Sorry for the overload, I blame it on quarantine.
Thank you and best wishes


Sent from Yahoo Mail for iPhone


On Thursday, April 9, 2020, 6:20 PM, Sarah Guthals <notifications@github.com> wrote:




@sguthals approved this pull request.

You did figure it out!

**Comment by sguthals** [comment]
Definitely don't have to apologize! That's the point of this repository, to practice! :D 

Stay healthy <3

**Pull request merged**